### PR TITLE
Changed yolo-v4-tf model to tf2

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

File: models/public/yolo-v4-tf/pre-convert.py [hunk 1]
```diff
@@ -35,11 +37,11 @@ def main():
...
-    subprocess.run([sys.executable, '--',
-        str(args.input_dir / 'keras-YOLOv3-model-set/tools/model_converter/keras_to_tensorflow.py'),
-        '--input_model={}'.format(args.output_dir / 'yolo-v4.h5'),
-        '--output_model={}'.format(args.output_dir / 'yolo-v4.pb'),
-    ], check=True)
+    sys.path.append(str(args.input_dir))
+    nets = importlib.import_module('keras-YOLOv3-model-set.yolo4.models.layers')
+
+    model = tf.keras.models.load_model(str(args.output_dir / 'yolo-v4.h5'), custom_objects={'mish': nets.mish})
+    model.save(filepath=args.output_dir / 'yolo-v4.savedmodel')
```
Comment (IRDonch) [inline models/public/yolo-v4-tf/pre-convert.py:44]: I'm not sure this is needed. Try to just change the fourth parameter of the previous command to `yolo-v4.savedmodel`.

File: models/public/yolo-v4-tf/accuracy-check.yml [hunk 2]
```diff
@@ -1,64 +1,4 @@
 models:
-  - name: yolo-v4-tf
```
Comment (IRDonch) [inline models/public/yolo-v4-tf/accuracy-check.yml:2]: Is there no way to validate the SavedModel version through AC?

File: models/public/yolo-v4-tf/pre-convert.py (hunk 1 above)
Comment (Anna-Mironova) [reply models/public/yolo-v4-tf/pre-convert.py]: Done

File: models/public/yolo-v4-tf/accuracy-check.yml (hunk 2 above)
Comment (Anna-Mironova) [reply models/public/yolo-v4-tf/accuracy-check.yml:2]: validation in source framework returned

**Review by eaidova — APPROVED** [review]

**Review by IRDonch — APPROVED** [review]
